test/model/src/driver/kv_workload_generator.cpp

```diff
@@ -215,6 +216,23 @@ kv_workload_generator::assert_timestamps(
         stable = t;
     }
 
+    if (std::holds_alternative<operation::set_oldest_timestamp>(op)) {
```

Comment:
I have a question regarding lines above which manage the stable ts.
What happens in this scenario:

- Start WT, not ts are set
- Set oldest to 10
- Set stable to 9

The above will not trigger as we set stable for the first time, but it is lower than the oldest ts.

Reply:
I also added a similar check to the stable timestamp branch above in [cc5b2fb](https://github.com/wiredtiger/wiredtiger/pull/10427/commits/cc5b2fb1f8ed4c230e66945e028a36edeca3fab4).